libs/ui-lib-tests/cypress/views/customManifestsPage.ts

```diff
@@ -0,0 +1,35 @@
+export const customManifestsPage = {
+  getFileName: () => {
+    return cy.findByTestId('filename-0');
+  },
```

Comment:
`getFileName` and `getManifest1Id` do pretty much the same thing. Merge them into one function with an index parameter. Something like:

```suggestion
  getFileName: (index: number) => {
    return cy.findByTestId('filename-${index}`);
  },
```